acstools/focus_diverse_epsfs.py

```diff
@@ -0,0 +1,460 @@
+"""This module contains two functions that provide
+means to programmatically query the `ACS/WFC
+Focus-Diverse ePSF Generator <acspsf.stsci.edu>`_ API.
```

Comment:
https://acspsf.stsci.edu does not work. Is this a real URL?

Reply:
this URL will be set to live once the module is available on acstools

Reply:
OK, chicken and egg then. I guess we can merge ignoring the failing link check job.